edward2/tensorflow/generated_random_variables.py

```diff
@@ -77,9 +77,11 @@ def func(*args, **kwargs):
     # pylint: enable=g-doc-args
     sample_shape = kwargs.pop("sample_shape", ())
     value = kwargs.pop("value", None)
+    seed = kwargs.pop("seed", None)
```

Comment:
The seed additions seem useful outside the JointDistribution PR! Mind adding them as a separate pull request? That one can be approved much faster.